src/Controls/src/Core/Platform/Windows/Extensions/CollectionViewExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using WSetter = Microsoft.UI.Xaml.Setter;
+using WStyle = Microsoft.UI.Xaml.Style;
```

Comment:
'#nullable enabled' ? If it is the default, I am surprise we are getting no warnings.